nestjs-otel-lib-example/src/tracing.ts

```diff
@@ -0,0 +1,48 @@
+import { Metadata, credentials } from "@grpc/grpc-js";
+import { NodeSDK } from "@opentelemetry/sdk-node";
+import * as process from "process";
+import { NestInstrumentation } from "@opentelemetry/instrumentation-nestjs-core";
+import { Resource } from "@opentelemetry/resources";
+import { SemanticResourceAttributes } from "@opentelemetry/semantic-conventions";
+import { OTLPTraceExporter } from "@opentelemetry/exporter-trace-otlp-grpc";
+import * as dotenv from "dotenv";
+import { getNodeAutoInstrumentations } from "@opentelemetry/auto-instrumentations-node";
+
+dotenv.config();
+
+// For troubleshooting, set the log level to DiagLogLevel.DEBUG
+import { diag, DiagConsoleLogger, DiagLogLevel } from "@opentelemetry/api";
+diag.setLogger(new DiagConsoleLogger(), DiagLogLevel.DEBUG);
```

Comment:
I suspect people will copy-paste this file as is. So, let's keep it in the appropriate state for production, and have any debug suggestions commented out, okay?

```suggestion
// For troubleshooting, set the log level to DiagLogLevel.DEBUG
// import { diag, DiagConsoleLogger, DiagLogLevel } from "@opentelemetry/api";
// diag.setLogger(new DiagConsoleLogger(), DiagLogLevel.DEBUG);
```